device/bgpv4.yaml

```diff
@@ -8,49 +8,75 @@ components:
   schemas:
     Device.Bgpv4:
       description: >-
-        Emulated bgpv4 protocol
+        Emulated BGPv4 router and routes
       type: object
       required: [name]
       properties:
         name: '../common/common.yaml#/components/x-inline/UniqueName'
         as_number_2_byte:
+          description: >-
+            Autonomous system (AS) number of 2 byte
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         dut_as_number_2_byte:
+          description: >-
+            Autonomous system (AS) number of the BGP peer router (DUT)
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         as_number_4_byte:
+          description: >-
+            Autonomous system (AS) number of 4 byte
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         as_number_set_mode:
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         as_type:
-          description: >- 
-            The type of BGP autonomous system. 
-            External BGP (EBGP) is used for BGP links between two or more 
+          description: >-
+            The type of BGP autonomous system.
+            External BGP (EBGP) is used for BGP links between two or more
             autonomous systems.
             Internal BGP (IBGP) is used within a single autonomous system.
           type: string
           enum: [IBGP, EBGP]
         hold_time_interval:
+          description: >-
+            Number of seconds the sender proposes for the value of the Hold Timer
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         keep_alive_interval:
+          description: >-
+            Number of seconds between transmissions of Keep Alive messages by router
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         graceful_restart:
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         authentication:
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         ttl:
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
-        dut_ipv4_address:         
+        dut_ipv4_address:
           $ref: './patterns.yaml#/components/schemas/Device.Pattern'
         ipv4:
           $ref: './ipv4.yaml#/components/schemas/Device.Ipv4'
+        bgpv4routerange:
+          description: >-
+            Emulated BGPv4 route range
+          type: array
+          items:
+            $ref: '#/components/schemas/Device.Bgpv4RouteRange'
+        bgpv6routerange:
+          description: >-
+            Emulated bgpv6 route range
+          type: array
+          items:
+            $ref: '#/components/schemas/Device.Bgpv6RouteRange'
 
     Device.Bgpv4RouteRange:
       description: >-
-        Emulated bgpv4 route range
+        Emulated BGPv4 route range
       type: object
       required: [name]
       properties:
         name: '../common/common.yaml#/components/x-inline/UniqueName'
+        route_count:
```

Comment:
add descriptions for all properties except inline

Reply:
I have added them according to my understanding. I feel we need to think actual representation of aigp_metric and atomic_aggregate within Bgpv4/v6RouteRange